packages/flutter/lib/src/rendering/semantics.dart

```diff
@@ -476,39 +546,41 @@ class SemanticsNode extends AbstractNode {
   }
 }
 
+/// Exposes the [SemanticsNode] tree to the underlying platform.
 class SemanticsServer extends mojom.SemanticsServer {
   @override
   void addSemanticsListener(mojom.SemanticsListenerProxy listener) {
+    // TODO(abarth): We should remove the listener when this pipe closes.
```

Comment:
oh actually the bug is about something related, namely that we should shut down semantics processing while we have no listeners. but similar idea.